reporter/src/funTest/assets/static-html-reporter-test-expected-output.html

```diff
@@ -350,8 +350,8 @@ <h3 class="">Packages</h3>
           </tr>
         </thead>
         <tbody>
-          <tr class="ort-error ">
-            <td>1</td><td>Gradle:com.here.ort.gradle.example:lib:1.0.0</td><td></td><td><em>Detected Licenses:</em>
+          <tr class="ort-error " id="Gradle:com.here.ort.gradle.example:lib:1.0.0-pkg-1">
```

Comment:
Note to myself: I was wondering whether we'd need to use `String.fileSystemEncode()` here to encode special chars, but according to http://xahlee.info/js/html_allowed_chars_in_attribute.html we should be fine.